nevergrad/functions/automl/core.py

```diff
@@ -0,0 +1,114 @@
+# Copyright (c) Meta Platforms, Inc. and affiliates. All Rights Reserved.
+#
+# This source code is licensed under the MIT license found in the
+# LICENSE file in the root directory of this source tree.
+#
+# Trained policies were extracted from https://github.com/modestyachts/ARS
+# under their own license. See ARS_LICENSE file in this file's directory
+import os
+
+import nevergrad.common.typing as tp
+import numpy as np
+from nevergrad.functions.base import UnsupportedExperiment as UnsupportedExperiment
+
+from .. import base
+
+
+class AutoSKlearnBenchmark(base.ExperimentFunction):
+    def __init__(
+        self,
+        openml_task_id: int,
+        cv: int,
+        time_budget_per_run: int,
+        memory_limit: int,
+        scoring_func: str = "balanced_accuracy",
+        error_penalty: float = 1.0,
+        overfitter: bool = False,
+        random_state: tp.Optional[int] = None,
+    ) -> None:
+
+        if os.name != "nt":
```

Comment:
do the opposite, if os.name == "nt" then raise 
then no need of indent anymore

Reply:
Ok.